cfn-templates/cid-cfn.yml

```diff
@@ -4,28 +4,26 @@ Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
-          default: 'Prerequisites'
+          default: 'Common'
```

Comment:
'Common parameters' maybe?

Reply:
Not sure. All here are parameters. 

It is like naming 'CURBucketPathParameter'

Reply:
It's slightly different as it's name of the group/section, not individual parameter.

Reply:
From the other hand we have Technical parameters below, so why not